### DISCUSSION TITLE
DataBia nos EUA – R-Ladies São Paulo

### DISCUSSION THREAD

**Discussion by giscus** [R-Ladies-Sao-Paulo/blog#37] [General]
# DataBia nos EUA – R-Ladies São Paulo

https://rladies-sp.org/posts/2024-07-temperaturas-eua/

**Comment by beatrizmilz** [comment]
Bianca, você faz visualizações incríveis! Parabéns e obrigada por compartilhar conosco! :tada:

**Comment by amandarochaalves** [comment]
Que incrível, amei!!
